<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataArea</class>
 <widget class="QWidget" name="DataArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="udpRadioPushButton">
      <property name="text">
       <string>广播</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="clearPushButton">
      <property name="text">
       <string>清空数据</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openAlarmPushButton">
      <property name="text">
       <string>开启服务</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="closeSerivePushButton">
      <property name="text">
       <string>关闭服务</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>511</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,0,1,3,0,2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ipComboBox">
        <item>
         <property name="text">
          <string>192.168.1.105</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portComboBox">
        <item>
         <property name="text">
          <string>6901</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6902</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6903</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6904</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6905</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectPushButton">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="showDataTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
